.github/workflows/ci.yml

```diff
@@ -128,6 +128,23 @@ jobs:
       - name: Upload test coverage to Codecov
         uses: codecov/codecov-action@v1.0.12
 
+  window_bento_service_in_linux_container_tests:
```

Comment:
just add to the api_server_integration_tests, make it a matrix similar to the unit tests
```
      matrix:
        os: [ubuntu-latest, macos-latest, windows-latest]
```